### PR TITLE
User redirectd to instructions screen if then have account.

### PR DESCRIPTION
**Pull request by mfduar8766**
@smuntean @iangornall

### CODE REVIEW

File: public/js/login.js [hunk 1]
```diff
@@ -58,6 +56,40 @@ let loginUser = (event) => {
...
+    return true;
+  }
+  return false;
+};
+
+let automaticSignIn = () => {
+  let checkUser;
+  if(localStorage.getItem("token")) {
+    checkUser = localStorage.getItem("token");
+    fetch(`/signedin`, {
```
Comment (iangornall) [inline public/js/login.js]: fetch request should be to '/api/signedin'.  I am getting a 404 error.

File: public/js/login.js [hunk 2]
```diff
@@ -58,6 +56,40 @@ let loginUser = (event) => {
...
+      }
+    })
+    .then(response => 
+        response.text())
+    .then(message => {
+      if(message === 'is user') {
+          loginForm.classList.remove("hidden");
+      }
+      else {
+        loginForm.reset();
```
Comment (iangornall) [inline public/js/login.js]: This logic does not make sense  The login form should be blank already, and getLevelData is called later.  Remove the else statement.  If the message === "is user", then showInstructions().

**Review by iangornall — CHANGES_REQUESTED** [review]
See comments.  The fetch request is broken and the logic needs changing.

File: public/js/login.js (hunk 1 above)
Comment (mfduar8766) [reply public/js/login.js]: i created a new route instead of /api

Comment (sarathrasher) [reply public/js/login.js]: Second this. authRouter only runs if the url starts with /api

File: public/js/login.js (hunk 2 above)
Comment (mfduar8766) [reply public/js/login.js]: Are you saying to do else {
 showLoginForm()
getLevelData()
}

File: server.js [hunk 3]
```diff
@@ -21,7 +22,9 @@ publicRouter.post('/login', loginUser);
 authRouter.use(validateToken);
 
 authRouter.get('/level_data/:id', getLevelData)
-
+authRouter.get('/signedin',(req,res) => {
+  res.send('is user');
```
Comment (sarathrasher) [inline server.js]: You're making the request through authRouter, not userId. Is userId necessary?

File: public/js/login.js (hunk 2 above)
Comment (iangornall) [reply public/js/login.js]: if the message === "is user", then showInstructions().
no else

**Review by sarathrasher — APPROVED** [review]
👍

File: public/js/login.js [hunk 4]
```diff
@@ -25,7 +25,7 @@ let createUser = (event) => {
   })
   .then(response => {
     loginForm.reset()
-    console.log(response);
+    getLevelData(level);
```
Comment (iangornall) [inline public/js/login.js]: remove the getLevelData

**Review by iangornall — CHANGES_REQUESTED** [review]

**Review by iangornall — APPROVED** [review]

**Pull request merged**